app/components/task-item.js

```diff
@@ -0,0 +1,23 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  init() {
+    this._super(...arguments);
+  },
+  didInsertElement() {
+    this.$().linkify({
+      validate: {
+        url: function (value) {
+          return /^(http|ftp)s?:\/\//.test(value);
+        },
+      },
+      formatHref: function (href, type) {
```

Comment:
This can also be an arrow function. Doesn't have much benefit besides consistency, but that's pretty important.